cmd/kube-apiserver/app/options/options.go

```diff
@@ -149,6 +152,7 @@ func (s *ServerRunOptions) Flags() (fss cliflag.NamedFlagSets) {
 	s.EgressSelector.AddFlags(fss.FlagSet("egress selector"))
 	s.Admission.AddFlags(fss.FlagSet("admission"))
 	s.Metrics.AddFlags(fss.FlagSet("metrics"))
+	s.Logs.AddFlags(fss.FlagSet("logging-format"))
```

Comment:
```suggestion
	s.Logs.AddFlags(fss.FlagSet("logs"))
```
We register a flag group for logs, maybe list of flags will grow in future.

Reply:
thanks! updated!